webapp/src/app.tsx

```diff
@@ -4142,6 +4143,51 @@ export class ProjectView
         return { categories };
     }
 
+    getBlocksWithId(blockId: string): BlockDefinition[] {
+        const matches: BlockDefinition[] = [];
+        for (const advanced of [false, true]) {
```

Comment:
Can you explain what the `advanced` is doing?

Reply:
When getting toolbox categories with the existing getToolboxCategories function, you can either get the "Advanced" categories (i.e. the ones under the Advanced header) or the non-advanced ones. This is just a way to check both.

Reply:
Sweet, thanks!